x/tokenfactory/keeper/keeper_test.go

```diff
@@ -0,0 +1,76 @@
+package keeper_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/suite"
+	"github.com/tendermint/tendermint/crypto/ed25519"
+
+	"github.com/neutron-org/neutron/testutil"
+	"github.com/neutron-org/neutron/x/tokenfactory/keeper"
+	"github.com/neutron-org/neutron/x/tokenfactory/types"
+)
+
+type KeeperTestSuite struct {
+	testutil.IBCConnectionTestSuite
+
+	TestAccs    []sdk.AccAddress
+	QueryHelper *baseapp.QueryServiceTestHelper
+
+	queryClient types.QueryClient
+	msgServer   types.MsgServer
+	// defaultDenom is on the suite, as it depends on the creator test address.
+	defaultDenom string
```

Comment:
I cannot really understand what "default denom is on the suite" is supposed to mean, please rewrite this comment